_config.yml

```diff
@@ -62,7 +62,7 @@ footer:
     - text: 'FOIA information'
       href: '/foia/'
     - text: 'Plain Writing Act'
-      href: '#'
+      href: 'http://www.plainlanguage.gov/pllaw/'
```

Comment:
Um, when I googled "plain writing act" this was the first hit, and it's on a .gov domain, so I figured it's probably what we want to link to?